src/main/java/com/spotinst/sdkjava/model/api/gcp/ApiElastigroupScaleDownResponseGcp.java

```diff
@@ -0,0 +1,63 @@
+package com.spotinst.sdkjava.model.api.gcp;
+
+import com.fasterxml.jackson.annotation.JsonFilter;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.spotinst.sdkjava.client.rest.IPartialUpdateEntity;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonFilter("PartialUpdateEntityFilter")
+public class ApiElastigroupScaleDownResponseGcp implements IPartialUpdateEntity {
+
+    @JsonIgnore
+    private Set<String>                                     isSet;
+    private List<ApiElastigroupScaleDownVictimPreemptiblesListGcp>  victimPreemptiblesList;
```

Comment:
Change class name from ApiElastigroupScaleDownVictimPreemptiblesListGcp to ApiElastigroupScaleDownVictimPreemptiblesGcp